sdk/lib/3rdparty/libwine/debug.c

```diff
@@ -413,26 +476,68 @@ static int default_dbg_vprintf( const char *format, va_list args )
 
 
 /* default implementation of wine_dbg_vlog */
-static int default_dbg_vlog( enum __wine_debug_class cls, struct __wine_debug_channel *channel,
-                             const char *file, const char *func, const int line, const char *format, va_list args )
+/* Wine format */
+static int winefmt_default_dbg_vlog( enum __wine_debug_class cls, struct __wine_debug_channel *channel,
+                                     const char *file, const int line, const char *func, const char *format, va_list args )
+{
+    int ret = 0;
+
+    if (TRACE_ON(pid))
+        ret += wine_dbg_printf( "%04x:", HandleToULong(NtCurrentTeb()->ClientId.UniqueProcess) );
+    ret += wine_dbg_printf( "%04x:", HandleToULong(NtCurrentTeb()->ClientId.UniqueThread) );
+
+    if (cls < sizeof(debug_classes)/sizeof(debug_classes[0]))
+        ret += wine_dbg_printf( "%s:%s:%s ", debug_classes[cls], channel->name, func );
+    if (format)
+        ret += funcs.dbg_vprintf( format, args );
+    return ret;
+}
+/* ReactOS format (default) */
+static int rosfmt_default_dbg_vlog( enum __wine_debug_class cls, struct __wine_debug_channel *channel,
+                                    const char *file, const int line, const char *func, const char *format, va_list args )
 {
     int ret = 0;
 
     if (TRACE_ON(tid))
-        ret += wine_dbg_printf("%04x:", HandleToULong(NtCurrentTeb()->ClientId.UniqueThread));
+        ret += wine_dbg_printf( "%04x:", HandleToULong(NtCurrentTeb()->ClientId.UniqueThread) );
```

Comment:
Why change whitespace?

Reply:
This makes it in par with the wine code this originally came from.